x-pack/test/functional/apps/security/field_level_security.ts

```diff
@@ -123,6 +125,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     after(async function () {
       // NOTE: Logout needs to happen before anything else to avoid flaky behavior
+      await security.testUser.restoreDefaults();
       await PageObjects.security.forceLogout();
```

Comment:
For consistency and to avoid possible flakiness, could you move this `restoreDefaults()` to the end??

```diff
diff --git a/x-pack/test/functional/apps/security/field_level_security.ts b/x-pack/test/functional/apps/security/field_level_security.ts
index 49d7e8e5245..4034f955429 100644
--- a/x-pack/test/functional/apps/security/field_level_security.ts
+++ b/x-pack/test/functional/apps/security/field_level_security.ts
@@ -125,11 +125,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     after(async function () {
       // NOTE: Logout needs to happen before anything else to avoid flaky behavior
-      await security.testUser.restoreDefaults();
       await PageObjects.security.forceLogout();
       await kibanaServer.importExport.unload(
         'x-pack/test/functional/fixtures/kbn_archiver/security/flstest/index_pattern'
       );
+      await security.testUser.restoreDefaults();
     });
   });
 }
```

Reply:
Yep. Sounds good.